.travis.yml

```diff
@@ -17,22 +17,40 @@ env:
   matrix:
     - DRIVER="xdebug" DEPENDENCIES="high"
     - DRIVER="phpdbg" DEPENDENCIES="high"
+    - DRIVER="pcov"   DEPENDENCIES="high"
     - DRIVER="xdebug" DEPENDENCIES="low"
     - DRIVER="phpdbg" DEPENDENCIES="low"
+    - DRIVER="pcov"   DEPENDENCIES="low"
   global:
     - DEFAULT_COMPOSER_FLAGS="--no-interaction --no-ansi --no-progress --no-suggest"
 
 before_install:
+  - composer config github-oauth.github.com ${GH_TOKEN}
   - composer self-update
   - composer clear-cache
+  - export COMPOSER_ROOT_VERSION=6.1.99
```

Comment:
Is this still needed?

Reply:
I think this one is still needed, because we still have a circular dep, phpunit depends php-code-coverage